Formula/clickhouse.rb

```diff
@@ -0,0 +1,88 @@
+class Clickhouse < Formula
+  desc "Free analytics DBMS for big data with SQL interface"
+  homepage "https://clickhouse.com"
+  url "https://github.com/ClickHouse/ClickHouse/releases/download/v21.11.1.8636-prestable/ClickHouse_sources_with_submodules.tar.gz"
+  sha256 "e95576f1b9f557ae7aa808fbe8959f14fbc86981d809d21e49eb65fe4ebf1142"
+  license "Apache-2.0"
+  head "https://github.com/ClickHouse/ClickHouse.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "gawk" => :build
+  depends_on "gettext" => :build
+  depends_on "libtool" => :build
+  depends_on "ninja" => :build
+  depends_on "perl" => :build
+  depends_on "python@3.9" => :build
+  depends_on "llvm" => "13"
```

Comment:
```suggestion
  depends_on "llvm"
```
`=> "13"` doesn't do anything. Do you link with `libLLVM`?

Reply:
Do you link with `libLLVM`?

Reply:
No. Not in macOS builds... Will change that to `=> :build`.

Reply:
Changed.